client/allocrunner/taskrunner/plugin_supervisor_hook.go

```diff
@@ -317,7 +363,7 @@ func (h *csiPluginSupervisorHook) registerPlugin(client csi.CSIPlugin, socketPat
 			Options: map[string]string{
 				"Provider":            info.Name, // vendor name
 				"MountPoint":          h.mountPoint,
-				"ContainerMountPoint": h.task.CSIPluginConfig.MountDir,
+				"ContainerMountPoint": "/local/csi",
```

Comment:
A task driver without filesystem isolation never creates the mount, so it's a no-op. (Note this is an exotic case because the spec requires that plugins are shipped as OCI containers)